app/main.py

```diff
@@ -16,4 +16,22 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+collection = [lucky_number,
+              pi,
+              one_is_a_prime_number,
+              name,
+              my_favourite_films,
+              profile_info,
+              marks,
+              collection_of_coins]
+
+mutable_types = (set, dict, list)
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+for item in collection:
+    if isinstance(item, mutable_types):
+        sorted_variables["mutable"].append(item)
+    else:
+        sorted_variables["immutable"].append(item)
```

Comment:
There is no need to create an additional logic in this task, just define and populate the `sorted_variables` variable with required variables as it is done in the task description. Do not create the `collection` variable as well for this task.

Reply:
[ArsenPidhoretskyi], asked to rename variables and change approach so i left logic in this task